exporter/clickhouselogsexporter/config.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clickhouselogsexporter
+
+import (
+	"errors"
+
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	"go.uber.org/multierr"
+)
+
+// Config defines configuration for ClickHouse exporter.
+type Config struct {
+	config.ExporterSettings        `mapstructure:",squash"`
+	exporterhelper.TimeoutSettings `mapstructure:",squash"`
+	exporterhelper.RetrySettings   `mapstructure:"retry_on_failure"`
+	// QueueSettings is a subset of exporterhelper.QueueSettings,
+	// because only QueueSize is user-settable.
+	QueueSettings QueueSettings `mapstructure:"sending_queue"`
+
+	// DSN is the ClickHouse server Data Source Name.
+	// For tcp protocol reference: [ClickHouse/clickhouse-go#dsn](https://github.com/ClickHouse/clickhouse-go#dsn).
+	// For http protocol reference: [mailru/go-clickhouse/#dsn](https://github.com/mailru/go-clickhouse/#dsn).
+	DSN          string `mapstructure:"dsn"`
+	DatabaseName string // `mapstructure:"database_name"` // dont read from config
+
+	LogsTableName string // `mapstructure:"logs_table_name"` // dont read from config
```

Comment:
Why not remove them here if not read from the config otherwise it'll be confusing? the names can be moved from here `clickhouseLogsExporter` writer?

Reply:
right now using this is used by createDefaultConfig, are you suggesting creating constants for them ?

Reply:
It was confusing why they existed here if it's values provided via config are not considered.

Reply:
yeah